### DISCUSSION TITLE
Add [Create empty entry in Calibre] button from Holds tab

### DISCUSSION THREAD

**Discussion by rhyddic** [sgmoore/libby-calibre-plugin#3] [Ideas]
First off, THANK YOU so much for taking on this project. Great job so far!  I appreciate the existing create empty entry button on the Loans tab and would find it as, if not more useful from the Holds tab. I, personally, use color coded rows for books in various states, such as borrowed, finished reading, on-hold, physical copy on hand, advanced reader copy, etc. Ideally, I would like to be able to easily add empty entries for my books on hold and automatically add a custom tag, such as "On Hold" or "Libby Hold" or whatever. Thanks again!

**Comment by sgmoore** [comment]
This feature has now been implemented in the new beta release 

See https://github.com/sgmoore/libby-calibre-plugin/discussions/8 for more details
